setup.py

```diff
@@ -39,13 +39,16 @@
     package_data={"smclarify": data_files},
     long_description=LONG_DESCRIPTION,
     long_description_content_type="text/markdown",
+    python_requires=">=3.7",
     install_requires=INSTALL_REQUIRES,
     extras_require=EXTRAS_REQUIRE,
     classifiers=[
         "Development Status :: 1 - Planning",
         "Intended Audience :: Developers",
         "Natural Language :: English",
         "Programming Language :: Python",
-        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
```

Comment:
let's remove 3.7 if we are not testing it?

Reply:
removed the 3.7 version from the documentation.